src/bika/lims/browser/fields/uidreferencefield.py

```diff
@@ -155,18 +155,39 @@ def get(self, context, **kwargs):
         :return: object or list of objects for multiValued fields.
         :rtype: BaseContent | list[BaseContent]
         """
-        value = StringField.get(self, context, **kwargs)
-        if not value:
-            return [] if self.multiValued else None
+        uids = StringField.get(self, context, **kwargs)
+        if not self.multiValued:
+            uids = [uids]
```

Comment:
better make this check `if not isinstance(uids, list): uids = [uids]`.

Reply:
Resolved with https://github.com/senaite/senaite.core/pull/2212/commits/45ed57f72cd0b470a57f7ba49b8313a93d5d2066